03_API_Reference/01_Ethereum_Provider.md

```diff
@@ -133,31 +319,31 @@ ethereum.sendAsync({
 })
 ```
 
-### ethereum.autoRefreshOnNetworkChange
+## Properties
 
-When the network is changed, MetaMask will reload any pages that have made requests to the provider. This automatic reload behavior will be removed in a future release of MetaMask, but in the meantime it can be disabled with this flag.
+Useful for knowing whether `window.ethereum` is MetaMask, but not much else.
```

Comment:
Was this left here by mistake? It sounds like it belongs under the `isMetaMask` header